src/components/Nav.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+
+function NavBar() {
+  return (
+    <nav>
+      <Link to="/articles">Home</Link>
+      <Link to="/articles/coding">Coding</Link>
+      <Link to="/articles/cooking">Cooking</Link>
+      <Link to="/articles/football">Football</Link>
```

Comment:
This is hardcoded, if in the future we need to add more categories, you have to update your whole page. We should instead fetch all the possible categories and map out an array of links from that. 

Furthermore, this is not very dry code. React router allows us to deal with parametric endpoints using /articles/:topic which means that coding, cooking, and football could all go to the same page and based on the topic in the url, using the useParams() hook, we can fetch whichever topic has been requested from the api. 

Look at the notes for more on useParams and React Router please